frontend/components/ProjectList.js

```diff
@@ -5,6 +5,7 @@ import { useQuery } from '@apollo/react-hooks';
 import withApollo from '../lib/apollo';
 import { CardText, CardTitle, Col, Row } from "reactstrap";
 import Project from "../components/Project";
+const apiUrl = process.env.API_URL || 'http://localhost:1337';
```

Comment:
Setting apiUrl instead of localhost, as im developing off a server. Would be great if could set this once globally and import everywhere.